cmd/timoni/main.go

```diff
@@ -152,3 +155,33 @@ func addKubeConfigFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().StringVarP(kubeconfigArgs.Namespace, "namespace", "n", *kubeconfigArgs.Namespace, "The the namespace scope for the operation.")
 	cmd.RegisterFlagCompletionFunc("namespace", completeNamespaceList)
 }
+
+func getCurrentKubeconfigPath() string {
+	defaultPath := path.Join(homedir.HomeDir(), ".kube", "config")
```

Comment:
I think we can set `defaultPath := ""`. The `cli-runtime` package that we use in Timoni has the home dir logic when the client is initialised with an empty kubeconfig path, so we don't need to import `k8s.io/client-go/util/homedir`.

Reply:
Done, please take a look.